rpc/internal/system_metrics_service.go

```diff
@@ -48,14 +50,30 @@ func (s *systemMetricsService) AddSystemMetrics(ctx context.Context, data *Syste
 	systemMetrics := &model.SystemMetrics{ID: data.Id}
 	systemMetrics.Setup(s.env)
 	if err := systemMetrics.Find(ctx); err != nil {
+		grip.Error(message.WrapError(err, message.Fields{
```

Comment:
maybe add id here? or task id?

Reply:
Done. The task ID isn't available here, so I just used the document ID.